AzureMonitorAgent/ama_tst/AMA-Troubleshooting-Tool.md

```diff
@@ -0,0 +1,37 @@
+# Troubleshooting Tool for Azure Monitor Linux Agent
```

Comment:
Can the tool be part of ama agent and be installed as part of ama extension install?  I assume this is just temporary solution till then?

Reply:
After giving it a second thought, if you are going to put the tool tgz on github, can you also add tool version info to the log when the tool is launched? Either add the version to the collector py in the header as a const and print that in the tool output, or create a version file? It will be hard to figure out what version of the tool css or customer is using in case we find issues with the tool when going over the logs.

Reply:
@NarineM the tool is eventually planned to be installed with AMA, this is just temporary. I like the idea of having a version number though for the troubleshooting tool, added this to the ama_troubleshooter.sh file to be printed upon startup.